### ISSUE TITLE
Current maintainer / official fork?

### ISSUE THREAD

**Issue by mkilling** [SecureBrain/ruby_apk#33]
I assume that this repo is no longer being maintained. Is there an 'official' fork?

If not, my team might be interested in becoming the maintainer of this gem. I have no idea how the process to transfer the ownership would work, though.

**Comment by amias-channer** [comment]
I could potentially help here as well

**Comment by shanempope** [comment]
It looks like this person updated it to ruby_zip 1.0 and published the gem: https://rubygems.org/gems/ruby_android/versions/0.7.7
Pull request: https://github.com/SecureBrain/ruby_apk/pull/27
Fork: https://github.com/tajchert/ruby_apk

**Comment by tajchert** [comment]
Yes as I needed it in 2015 and to work with mimap folder. However I'm not a ruby dev myself so would love second maintainer to my repo if anybody wants to continue work on this -> feel free to contact me.
